pkg/setup/setup.go

```diff
@@ -124,7 +124,9 @@ func (s *Setup) Configure(flags *Flags, clusterURL string, attach bool) (*config
 		httpClient := httpclient.New(cluster.URL(), httpOpts...)
 		var err error
 		acsToken, err = s.loginFlow.Start(flags.loginFlags, httpClient)
-		if err != nil {
+		if err == login.ErrAuthDisabled {
+			s.logger.Warn("This cluster does not have authorization enabled. Skipping.")
```

Comment:
> This cluster does not have authentication enabled. Skipping login.

Reply:
Thanks, @bamarni. It's important to distinguish authentication and authorization. I still however prefer something in the lines of "This cluster does not require authenticated requests".